vercel/data_source_secure_compute_network.go

```diff
@@ -0,0 +1,312 @@
+func (r *secureComputeNetworkDataSource) Schema(_ context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Description: `
+Provides information about an existing Vercel Secure Compute Network.
+
+This data source allows you to retrieve details about a Secure Compute Network by its name and optional team ID.
+`,
+		Attributes: map[string]schema.Attribute{
+			"name": schema.StringAttribute{
+				Description: "The name of the Secure Compute Network configuration.",
+				Required:    true,
+			},
+			"team_id": schema.StringAttribute{
+				Description: "The ID of the Vercel team the Secure Compute Network belongs to. " +
+					"If omitted, the provider will use the team configured on the provider or the user's default team.",
+				Optional: true,
+			},
+			"id": schema.StringAttribute{
+				Description: "The unique identifier of the Secure Compute Network.",
+				Computed:    true,
+			},
+			"dc": schema.StringAttribute{
+				Description: "The data center (region) associated with the Secure Compute Network.",
+				Computed:    true,
+			},
+			"project_ids": schema.SetAttribute{
+				Description: "A list of Vercel Project IDs connected to this Secure Compute Network.",
+				Computed:    true,
+				ElementType: types.StringType,
+			},
+			"projects_count": schema.Int64Attribute{
+				Description: "The number of Vercel Projects connected to this Secure Compute Network.",
+				Computed:    true,
+			},
+			"peering_connections_count": schema.Int64Attribute{
+				Description: "The number of peering connections established for this Secure Compute Network.",
+				Computed:    true,
+			},
+			"cidr_block": schema.StringAttribute{
+				Description: "The CIDR block assigned to the Secure Compute Network.",
+				Computed:    true,
+			},
+			"availability_zone_ids": schema.SetAttribute{
+				Description: "A set of AWS Availability Zone IDs where the Secure Compute Network resources are deployed.",
+				Computed:    true,
+				ElementType: types.StringType,
+			},
+			"version": schema.StringAttribute{
+				Description: "The current version identifier of the Secure Compute Network configuration.",
+				Computed:    true,
+			},
+			"configuration_status": schema.StringAttribute{
+				Description: "The operational status of the Secure Compute Network (e.g., 'ready', 'create_in_progress').",
+				Computed:    true,
+			},
+			"aws": schema.SingleNestedAttribute{
+				Description: "AWS configuration for the Secure Compute Network.",
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"account_id": schema.StringAttribute{
+						Description: "The AWS account ID.",
+						Computed:    true,
+					},
+					"region": schema.StringAttribute{
+						Description: "The AWS region.",
+						Computed:    true,
+					},
+					"elastic_ip_addresses": schema.SetAttribute{
+						Description: "A list of Elastic IP addresses.",
```

Comment:
```suggestion
						Description: "A list of Public IP addresses.",
```

Also, we should move this out of the AWS struct.